lib/rules/keyword-spacing.js

```diff
@@ -59,6 +59,18 @@ function isCloseParenOfTemplate(token) {
     return token.type === "Template" && TEMPLATE_CLOSE_PAREN.test(token.value);
 }
 
+/**
+ * Checks if the given token is an `of` token or not.
+ *
+ * `of` is not a Keyword token.
+ *
+ * @param {Token} token - The token to be check.
+ * @returns {boolean} `true` if the token is an `of` token.
+ */
+function isOfToken(token) {
+    return token.value === "of";
```

Comment:
So, `for (const x of /*of*/ iterable) {}` triggers false positive. Wow, it's a bug in the current master as well. I will fix it.